packages/terra-alert/src/Alert.scss

```diff
@@ -0,0 +1,149 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-Alert {
```

Comment:
Updated implementation to use CSS Modules and updated the branch.  Please take a look.